apps/src/code-studio/pd/professional_learning_landing/csPrinciplesAndDiscoveriesSection.jsx

```diff
@@ -1,16 +1,64 @@
 /** Section for CS Principles and CS Discoveries */
 
 import React from 'react';
+import {TwoPartBanner, imageStyle, textStyle, leftStyle, rightStyle} from './twoPartBanner';
+import {Button} from 'react-bootstrap';
 
 const CsPrinciplesAndDiscoveriesSection = React.createClass({
   propTypes: {
+    coursesCompleted: React.PropTypes.arrayOf(React.PropTypes.string),
+    lastWorkshopSurveyUrl: React.PropTypes.string
+  },
 
+  renderCompletedCourseBox() {
+    if (this.props.lastWorkshopSurveyUrl) {
+      return (
+        <div id="cspStartSurvey">
+          <h3>
+            Share your feedback
+          </h3>
+          <p>
+            Thank you for completing a Code.org professional learning owrkshop! Please complete this survey so that we
+            may improve your experience.
+          </p>
```

Comment:
What about after the survey is completed? This is exposed through `Pd::Enrollment.completed_survey_id` after ~ a minute, or immediately in `PEGASUS_DB[:forms].where(kind: 'PdWorkshopSurvey', source_id: enrollment.id)`